client/TopDetails.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components';
+
+var Name = styled.div`
+  display: inline-block;
+  margin: 0;
+  line-height: 160%;
+  font-family: 'Arial';
+  font-size: 20px;
+  font-weight: 700;
+  color: #363636;
+  max-width: calc(100% - 80px);
+`;
+
+var Wishlist = styled.div`
+  cursor: pointer;
+  display: inline-block;
+  line-height: 32px;
+  float: right;
+  font-size: 12px;
+  font-weight: bold;
+  color: #1428a0;
+`;
+
+var ProductCode = styled.div`
+  font-family: 'Arial';
+  font-size: 10px;
+  line-height: 240%;
+  font-weight: 700;
+  margin-right: 15px;
+  color: #a6a6a6;
+`;
+
+var Ratings = styled.div`
+  display: inline-block;
+  font-size: medium;
+  line-height: 1;
+  text-align: start;
+  color: black;
+`;
+
+var Share = styled.div`
+  cursor: pointer;
+  display: inline-block;
+  color #0077C8;
+  font-size: 12px;
+  vertical-align: 5px;
+`;
```

Comment:
Try to work toward consistency  and brevity in your styling. For example: Use either 'bold' or '700' for font-weight but not a mix of the two (since they are equivalent), use the same units consistently for font-size, etc.  Take advantage of inheritance so you don't have to, for example, re-assign font-family in multiple places if that's going to be the typeface you consistently use.